reconcile/cli.py

```diff
@@ -3333,6 +3333,16 @@ def acs_policies(ctx):
         ctx=ctx.obj,
     )
 
+@integration.command(short_help="Automate Deadmanssnitch Creation/Deletion")
+@click.pass_context
+def deadmanssnitch_automation(ctx):
```

Comment:
NIT: can we just name this `deadmansnitch`? Same for the file `reconcile/deadmanssnitch_automation.py`. All this is about automation, no need to mention it ;-)

Reply:
@patjlm updated the integratiion name to `deadmanssnitch`